src/test/java/com/labsynch/labseer/domain/AnalysisGroupDataOnDemand.java

```diff
@@ -1,7 +1,165 @@
 package com.labsynch.labseer.domain;
 
-import org.springframework.roo.addon.dod.RooDataOnDemand;
+import java.security.SecureRandom;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Random;
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+import org.springframework.beans.factory.annotation.Configurable;
+import org.springframework.stereotype.Component;
 
-@RooDataOnDemand(entity = AnalysisGroup.class)
+@Configurable
```

Comment:
It's been a very long time so I don't expect these to succeed, but have you tried running the Junit tests?
I am more interested in whether they compile / start running successfully, but not interested in the tests actually passing.

Reply:
Verified that unit tests still compile and run.  Most of them fail because they don't have a properly seeded database to start with.

To get them to work I had to:

- Java unit test vscode extension requires junit 4.12, we are on 4.11
- Added this line to the  applicationContext.xml file:
```
     <context:property-placeholder ignore-resource-not-found="true"  location="file:///${HOME}/.acas/conf.properties" order="1" ignore-unresolvable="true" />
```
- I grabbed the compiled conf.properties from inside the docker container and replace `db` with `localhost`.  I put the resulting file in ~/.acas/conf.properties on my local system
- Started up acas docker setup using default indigo chemistry package
- I ran `mvn clean compile war:war -P indigo`
- Then I ran unit test via vs code java plugin

A number of them succeed but most fail because they are expecting the database to be populated with very specific data to run:
![Screen Shot 2022-04-29 at 1 41 16 PM](https://user-images.githubusercontent.com/868119/166066391-f8fbe3bd-d5e3-4c77-807e-45356547a7b3.png)